controllers/podiochaos/controller.go

```diff
@@ -105,7 +110,7 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	pbClient, err := chaosdaemon.NewChaosDaemonClient(ctx, r.Client, pod)
 	if err != nil {
 		r.Recorder.Event(obj, "Warning", "Failed", err.Error())
-		return ctrl.Result{}, nil
+		return ctrl.Result{Requeue: true}, nil
```

Comment:
Why do you prefer to using `return ctrl.Result{Requeue: true}, nil`, not `return ctrl.Result{}, err`? 🤔

Reply:
Because `return ctrl.Result{}, err` will stop the worker, and as we have only one worker per controller, it will stop to handle new request until the backoff ends.

Reply:
Workers are controlled by `wait.Until(c.worker, c.JitterPeriod, stop)`, so I think is not a big deal, just 1s delay for raising new worker. 🤔

I prefer to return err, because it is more simple.

Reply:
I also prefer to return an error, as there is a ready-made error.

Reply:
No. I think the `error` should be an internal error of the controller, but not a mistake from user input (like target pod hasn't set up...). We should handle the mistakes from user input in the controller. The way how we handle them is to requeue.

Reply:
1 second for chaos applying is a really big latency (and it could be more, as it retrys multiple times), which I don't think is acceptable.

Reply:
OK, got that.